app/assets/javascripts/components/state_persistence_component.js

```diff
@@ -1,55 +1,55 @@
 // @flow
```

Comment:
Now, the path / file name does not seem right, since its content is not a react component anymore. Maybe move it to libs?